README.md

```diff
@@ -51,6 +51,37 @@ Dex's main production use is as an auth-N addon in CoreOS's enterprise Kubernete
 
 More docs for running dex as a Kubernetes authenticator can be found [here](Documentation/kubernetes.md).
 
+## Connectors
+
+When a user logs in through dex, the user's identity is usually stored in another user-management system: a LDAP directory, a GitHub org, etc. Dex acts a shim between a client app and the upstream identity provider. The client only needs to understand OpenID Connect to query dex, while dex implements an array of protocols for querying other user-management systems.
+
+![](Documentation/img/dex-flow.png)
+
+A "connector" is a strategy used by dex for authenticating a user against another identity provider. Dex implements connectors that target specific platforms such as GitHub, LinkedIn, and Microsoft as well as established protocols like LDAP and SAML.
+
+Depending on the connectors limitations in protocols can prevent dex from issuing [refresh tokens][scopes] or returning [group membership][scopes] claims. For example, because SAML doesn't provide a non-interactive way to refresh assertions, if a user logs in through the SAML connector dex won't issue a refresh token to its client. Refresh token support is required for clients that require offline access, such as `kubectl`.
+
+Dex implements the following connectors:
+
+| Name | supports refresh tokens | supports groups claim | status | notes |
+| ---- | ----------------------- | --------------------- | ------ | ----- |
+| [LDAP](Documentation/connectors/ldap.md) | yes | yes | stable | |
+| [GitHub](Documentation/connectors/github.md) | yes | yes | stable | |
+| [SAML 2.0](Documentation/connectors/saml.md) | no | yes | stable |
+| [GitLab](Documentation/connectors/gitlab.md) | yes | yes | beta | |
+| [OpenID Connect](Documentation/connectors/oidc.md) | yes | no ([#1065][issue-1065]) | beta | Includes Google, Salesforce, Azure, etc. |
+| [LinkedIn](Documentation/connectors/linkedin.md) | yes | no | beta | |
+| [Microsoft](Documentation/connectors/microsoft.md) | yes | no | beta | |
```

Comment:
I'm not sure the unconditional "no" to _supports groups claims_ is accurate. Should this reference the [docs caveat](https://github.com/coreos/dex/blob/master/Documentation/connectors/microsoft.md#caveats), maybe?

Reply:
Ah, you're right. It does support groups, my bad.